waagent

```diff
@@ -5909,8 +5916,14 @@ def main():
         else:
             args.append(a)
     global Config
-    Config = ConfigurationProvider()
+    Config = ConfigurationProvider(conf_file)
     
+    logfile = Config.get("Logs.File")
+    if logfile is not None:
+        myLogger.file_path = logfile
+    logconsole = Config.get("Logs.Console")
+    if logconsole is not None and logconsole.lower().startswith("y"):
```

Comment:
That step is for manual installation. In factor, most users will use rpm or dpkg to install the agent. For example, on SuSE, we can't overwrite the config file.
All I'm asking is to keep the change only for RancherOS and docker based distribution.